docs/src/modules/components/CodeEditor.js

```diff
@@ -18,9 +19,8 @@ const Wrapper = styled(MarkdownElement)(({ theme }) => ({
     marginTop: theme.spacing(0),
   },
   borderRadius: theme.shape.borderRadius,
-  outlineOffset: '-1px',
   '&:hover': {
-    outline: `2px solid ${
+    outline: `3px solid ${
```

Comment:
@siriwatknp IMO it's a bit bold, I didn't see it in other places of MUI

Reply:
I'll leave this to @danilo-leal to decide.

Reply:
I wrote about this in point 7. of https://github.com/mui/material-ui/pull/32107#pullrequestreview-951032470. I didn't see this comment before. So I agree with @nihgwu 👍 for 2px.